crates/shared/src/sources/uniswap_v3/pool_fetching.rs

```diff
@@ -451,7 +451,15 @@ impl Maintaining for UniswapV3PoolFetcher {
             self.events.run_maintenance(),
             self.checkpoint.update_missing_pools()
         );
-        result1.and(result2)?;
+        result1?;
+        // since failure in updating the missing pools is not critical for UniswapV3PoolFetcher maintenance
+        // and future liquidity fetch calls, then there is no need to return error
+        if let Err(err) = result2 {
+            tracing::warn!(
+                "UniswapV3PoolFetcher failed to update missing pools: {}",
+                err
+            );
+        }
```

Comment:
If I remember correctly, pool fetching works by:
- If the pool is missing, queue an update
- Do an update
- Get the updated pool info.

Is there some code in the pool fetching that expects the missing pool to be there after the update?

Reply:
No. Failure to update missing pools is safe. The only consequence is that we would need to wait for the next maintenance run for it to be updated. Also, if we try to fetch liquidity for tokens contained in the missing pool before the next maintenance run it will be a cache miss therefore not critical.

Reply:
Do we get old data or do we get no data? Old data might be a problem.

Reply:
We get no data.